scaleway/resource_object_bucket.go

```diff
@@ -48,6 +48,9 @@ func resourceScalewayObjectBucketCreate(d *schema.ResourceData, m interface{}) e
 	acl := d.Get("acl").(string)
 
 	s3Client, region, err := s3ClientWithRegion(d, m)
+	if err != nil {
```

Comment:
I added a return for this error, if it's something unwanted it has to be changed to `_` (err value is overwritten on the next row).